src/Broadcasting/src/GuardInterface.php

```diff
@@ -4,9 +4,15 @@
 
 namespace Spiral\Broadcasting;
 
+use Psr\Http\Message\ResponseInterface;
 use Psr\Http\Message\ServerRequestInterface;
 
 interface GuardInterface
 {
-    public function authorize(ServerRequestInterface $request): bool;
+    /**
+     * Authenticate a websocket connection request.
+     */
+    public function authorize(
+        ServerRequestInterface $request
+    ): ResponseInterface;
```

Comment:
Can we use DTO there? Like
success: bool
topics: string[]|string
response: ResponseInterface